buildroot/share/tests/BIGTREE_GTR_V1_0-tests

```diff
@@ -21,7 +21,9 @@ opt_set TEMP_SENSOR_7 1
 opt_set E0_AUTO_FAN_PIN PC10
 opt_set E1_AUTO_FAN_PIN PC11
 opt_set E2_AUTO_FAN_PIN PC12
-exec_test $1 $2 "BigTreeTech GTR 8 Extruders with Auto-Fan"
+opt_set X_DRIVER_TYPE TMC2208
+opt_set Y_DRIVER_TYPE TMC2130
+exec_test $1 $2 "BigTreeTech GTR 8 Extruders with Auto-Fan and Mixed TMC Drivers"
 
```

Comment:
I should expand one of these tests to include BLTOUCH to make sure we are compiling in the servo class.